src/Lens/Family/Complete.hs

```diff
@@ -0,0 +1,94 @@
+{-# LANGUAGE DefaultSignatures #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE TypeOperators     #-}
+{-# LANGUAGE CPP               #-}
+
+module Lens.Family.Complete
+    ( Full(..)
+    , GFull(..) -- TODO: Why isn't GEmpty exported from the other module?
```

Comment:
There's no particular reason I didn't expose `GEmpty`.  You can expose it if you want